mirage/config.ml

```diff
@@ -2,6 +2,14 @@ open Mirage
 
 let net = generic_stackv4 default_network
 
+let seed =
+  let doc = Key.Arg.info ~doc:"Seed for private key." ["seed"] in
+  Key.(create "seed" Arg.(opt string "180586" doc))
```

Comment:
Is this default seed safe? Seems like everyone generating the same key would be worrisome if thats what this does.

Reply:
I'm not entirely sure I can follow you.  It may be worth if you can elaborate on the threat model you have in mind.

From the README.md:
``You can run `awa_gen_key` to get a seed (to be passed to the unikernel) and a public key (which you need to enable access to the git repository).`` So, yes, you'll need an SSH public and private key pair if you plan to use git via ssh. You'll need to put the public key onto your git repository authorized keys. The private key will be generated from a seed at startup (take a look at `awa_gen_key` if you're interested - this generates a fresh seed and public/private key pair each time).

But anyways, I suspect removing the default seed here doesn't hurt...

Reply:
Yeah I didn't think through a threat model, just when I see a magic number next to 'private key' I was wondering if someone accidentally passes in an empty key it should error out. I also am not sure how the seed affects the key so there is definitely my own ignorance here.

Reply:
Thanks for pointing this out, I hope d000926 fixes this to your satisfaction :)